lib/opensusebasetest.pm

```diff
@@ -381,6 +371,35 @@ sub export_kde_logs {
             upload_logs $fn;
         }
         save_screenshot;
+    } elsif (check_var("DESKTOP", "gnome")) {
+        script_run("tar -jcv -f /tmp/gdm.tar.bz2  /home/bernhard/.cache/gdm");
+        upload_logs('/tmp/gdm.tar.bz2', failok => 1);
+    }
+
+    # check whether xorg logs exist in user's home, if yes, upload xorg logs
+    # from user's home instead of /var/log
+    my $log_path = '/home/*/.local/share/xorg/X*';
```

Comment:
Perhaps as a follow up but, wouldn't it be easier to simply do:
```
tar -cvjf Xorg-logs.tar.bz2 /home/*/.local/share/xorg
```

As it's done too for the ssdm session logs?

Reply:
Yes, but I thought to not change it too much for people that were already using this log.
I would prefer to make that change in a separate PR, so that if someone complains, we can revert just that easily.

Reply:
It will only do that if the directory exists. I wanted to do that based on the written files rather than on the detected DM/DE.
Less maintenance.